themes/tiddlywiki/vanilla/base.tid

```diff
@@ -135,6 +135,10 @@ input:not([type]) {
 	background: <<colour background>>;
 }
 
+input[type="checkbox"] {
+  vertical-align: middle;
+}
+
```

Comment:
@pmario, I tend to agree that it feels more natural to have a checkbox centered with the text.